scripts/ci/test_local_annotations.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Define the file path
+FILE="$PASH_TOP/../annotations/pash_annotations/annotation_generation/AnnotationGeneration.py"
+
+# First run the script normally, make sure that the optimization occurred by checking that the FIFOs were made
+$PASH_TOP/pa.sh $PASH_TOP/evaluation/intro/hello-world.sh -d 2 --local-annotations-dir ~/annotations/ --log_file "$PASH_TOP/pash.log"
```

Comment:
If you test on a script with a single pipeline, you can use this flag to return whether it succeeded or not `--assert_compiler_success`. The flag is also used in other tests so you can find how to use it instead of using grep :)